styles.css

```diff
@@ -0,0 +1,20 @@
+#nav-bar {
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+    font-family: 'Exo 2', sans-serif;
+    padding: 1em;
+    display: flex;
+    margin-top: 40px;
+  }
```

Comment:
Please use a class attribute for multiple elements.IDs are used in one element

Reply:
- [ ] You have done a great job to be polite when requesting for the changes 👏 . However, the student may not be able to understand why this should be done. Therefore i suggest that you provide a detailed explaination to why you should prefer a student to use the class selectors. 
- [ ] [OPTIONAL] You may also consider attaching the link which the student can refer to, to have a deep understanding 🥰 .